plugins/inventory/proxmox.py

```diff
@@ -443,7 +443,8 @@ def _get_vm_config(self, properties, node, vmid, vmtype, name):
     def _get_vm_status(self, properties, node, vmid, vmtype, name):
         ret = self._get_json("%s/api2/json/nodes/%s/%s/%s/status/current" % (self.proxmox_url, node, vmtype, vmid))
         properties[self._fact('status')] = ret['status']
-        properties[self._fact('qmpstatus')] = ret['qmpstatus']
+        if vmtype == 'qemu':
```

Comment:
@Thulium-Drake tnx for fixing this 

We should change this line to this. This is only applicable when we have all 3 options set.  Like on this line 
https://github.com/ansible-collections/community.general/blob/59c6fad7c728bb7cae93c39ecb2aaa813ca4974b/plugins/inventory/proxmox.py#L524

```
if self.get_option('want_facts') and vmtype == 'qemu' and self.get_option('qemu_extended_statuses'):
```

Reply:
Sounds like a good plan, fixed!